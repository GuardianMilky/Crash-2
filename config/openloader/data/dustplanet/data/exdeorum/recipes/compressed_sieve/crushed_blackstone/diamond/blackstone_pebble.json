{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/crushed_blackstone"},"mesh":"exdeorum:diamond_mesh","result":"exdeorum:blackstone_pebble","result_amount":{"type":"minecraft:binomial","n":35.0,"p":0.7}}