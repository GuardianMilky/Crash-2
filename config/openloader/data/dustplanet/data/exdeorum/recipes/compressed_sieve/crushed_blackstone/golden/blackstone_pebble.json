{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/crushed_blackstone"},"mesh":"exdeorum:golden_mesh","result":"exdeorum:blackstone_pebble","result_amount":{"type":"minecraft:binomial","n":35.0,"p":0.7}}